zer0-os/zOS
# refactor(dialog-manager): add logout dialog and remove renders

### What does this do?
-  add logout dialog to manager and remove render from messenger list

### Why are we making this change?
- refactoring / clean up.

### How do I test this?
- run tests as usual.
- run ui and trigger log out flow > check logout dialog renders. 

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
